ara/ui/templates/partials/host_results.html

```diff
@@ -145,6 +145,13 @@
                     {% else %}
                       {% include "partials/result_status_icon.html" with status=result.status %}
                     {% endif %}
+                    {% if result.task.tags | length == 0 %}
+                      <span class="badge badge-pill badge-secondary invisible" class="btn btn-secondary">
+                    {% else %}
+                      <span class="badge badge-pill badge-secondary" class="btn btn-secondary" data-toggle="tooltip"  data-placement="right" data-html="true" title="{% include 'partials/tags_list.html' with tags=result.task.tags %}">
+                    {% endif %}
+                      {{ result.task.tags | length }} tags
+                      </span>
```

Comment:
I was able to reproduce the issue I mentioned in my [last comment](https://github.com/ansible-community/ara/pull/281#issuecomment-891240781) locally and it's indeed tripping over the spaces before and after the pipe -- if there are no spaces it works. I can reproduce the issue with the local server running, no need to generate a static report.

This is surprising to me because there's plenty of places where we pipe templatetags with spaces around the pipe (like [here](https://github.com/ansible-community/ara/blob/7ce19a37c0c42cc0ad2b36b3ebed19bed99ff0df/ara/ui/templates/partials/playbook_results.html#L161-L162)) so I'm mildly confused at it -- it may have to do with being inside an if block ? I'll have learned something today.

Anyway, I suggest switching the condition around so we don't need to test for the length (an empty list is false):

```suggestion
                    {% if result.task.tags %}
                      <span class="badge badge-pill badge-secondary" class="btn btn-secondary" data-toggle="tooltip"  data-placement="right" data-html="true" title="{% include 'partials/tags_list.html' with tags=result.task.tags %}">
                    {% else %}
                      <span class="badge badge-pill badge-secondary invisible" class="btn btn-secondary">
                    {% endif %}
                      {{ result.task.tags | length }} tag{{ result.task.tags | pluralize }}
                      </span>
```

I've also added a ``| pluralize`` filter that automatically adds an "s" if there is >1 tag.